www/static/js/docs.js

```diff
@@ -17,19 +17,30 @@
 
 $(document).ready(function () {
 
-    function slugify(text) {
+    function slugifyLikeGitHub(originalText) {
+
+        var text = originalText;
+
+        // convert to lowercase
         text = text.toLowerCase();
 
-        // replace unaccepted characters with spaces
+        // replace whitespace characters with dashes
+        text = text.replace(/\s/g, '-');
+
+        // remove dashes at the start and end if they weren't there since the start
```

Comment:
If they weren't there from the start? What about these two cases:
```
"  -hello" => "--hello"
"   hello" => "hello"
```
Is that right? Should the first example work out that way with the extra dash? Seems like `trim()` should maybe get called before the first replace